content/blog/20250806_fluentd-v1.19.0-has-been-released.md

```diff
@@ -0,0 +1,122 @@
+You had to manually identify and delete the corrupted files.
+
+This release improves buffer corruption detection.
+When corruption is detected, the file is automatically moved to a backup directory, and startup errors are suppressed.
+
+### Enhanced metrics
+We have improved Fluentd's observability by enhancing its metrics capabilities:
+
+* `Input` plugin metrics are now enabled by default.
+  * Previously, you needed to explicitly set `enable_input_metrics` to `true`. From this release, it's collected by default. To disable, set `enable_input_metrics false`.
+* New metrics for `in_tail`:
+  * `tracked_file_count`: Number of files currently being tailed.
+* New metrics for `output` plugin:
+  * `write_secondary_count`: Cumulative count of records written to secondary output.
+  * `drop_oldest_chunk_count`: Cumulative count of chunks dropped when `drop_oldest_chunk` is used for `overflow_action`.
+
+### Performance improvement: switch from `yajl-ruby` to `json` gem
+Previously, we used `yajl-ruby` for high-performance JSON processing, such as writing buffer chunks and parsing with `parser_json`.
+
+However, in recent years, the standard Ruby json library has improved performance significantly. So we changed to use the `json` gem as the default JSON processor.
+
+### Zstandard (zstd) compression support
+We have added support for the Zstandard (zstd) compression format, which offers both high compression ratios and faster processing compared to gzip.
+
+* The following plugins now accept compress zstd:
+  * [buffer](https://docs.fluentd.org/configuration/buffer-section#buffering-parameters)
+  * [out_file](https://docs.fluentd.org/output/file#compress)
+  * [out_forward](https://docs.fluentd.org/output/forward#compress)
+
+**Note:** To use zstd with `out_forward`, the destination server should support it. You cannot use it with Fluent Bit or Fluentd versions older than v1.19.0.
+
+### TLS 1.3 support
+The `out_http` plugin now supports TLS 1.3, offering enhanced security.
+By default, TLS 1.2 is used. To enable TLS 1.3, set the [tls_version](https://docs.fluentd.org/output/http#tls_version) parameter as follows:
+
+```
+<match tag>
+  @type http
+  ...
+  tls_version TLSv1_3
+  ...
+</match>
+```
+
+### Other useful improvements
+* `in_forward` plugin:
+  * [skip_invalid_event](https://docs.fluentd.org/input/forward#skip_invalid_event) is now enabled by default to skip corrupted records.
+* `out_stdout` plugin:
+  * Added the [use_logger](https://docs.fluentd.org/output/stdout#use_logger) parameter. Setting `use_logger false` forces output to STDOUT regardless of Fluentd's own logging configuration. This is especially useful in container environments like Kubernetes.
+* `out_file` plugin:
+  * Added `symlink_path_use_relative` to allow relative paths in `symlink_path`.
+* System configuration:
+  * Added [forced_stacktrace_level](https://docs.fluentd.org/deployment/system-config#forced_stacktrace_level) to control the log level of stack traces.
```

Comment:
<img width="671" height="215" alt="image" src="https://github.com/user-attachments/assets/ee3987c9-ecf4-4f10-a09a-8cef3c694ce2" />

A bit funny rendering links.

Reply:
`[skip\_invalid\_event]` seems valid here.

Reply:
[\`skip_invalid_event\`] is also valid, but it breaks link style rendering.

Reply:
Fixed with https://github.com/fluent/fluentd-website/pull/386/commits/ed1a71c44180bc039fa609cd339298262b14e138